src/PageHeader/PageHeader.stories.tsx

```diff
@@ -0,0 +1,413 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {Button, IconButton, Breadcrumbs, Link, Text, StateLabel, BranchName} from '..'
+import {UnderlineNav} from '../UnderlineNav2'
+import Label from '../Label'
+import {
+  KebabHorizontalIcon,
+  GitBranchIcon,
+  CodeIcon,
+  GitPullRequestIcon,
+  PeopleIcon,
+  PencilIcon,
+  CommentDiscussionIcon,
+  CommitIcon,
+  ChecklistIcon,
+  FileDiffIcon,
+  ArrowRightIcon
+} from '@primer/octicons-react'
+import {OcticonArgType} from '../utils/story-helpers'
+
+import {PageHeader} from './PageHeader'
+import Hidden from '../Hidden'
+
+const meta: Meta = {
+  title: 'Drafts/Components/PageHeader',
+  parameters: {
+    layout: 'fullscreen',
+    controls: {expanded: true}
+  },
+  args: {
+    'ContextArea.hidden': false,
+    ParentLink: 'Previous Page',
+    'ParentLink.hidden': false,
+    'ContextBar.hidden': true,
+    'ContextAreaAction.hidden': true,
+    'BackButton.hidden': false,
```

Comment:
Now that I'm seeing this live, it seems to me having this as a `leadingAction` (since it's placed alongside `trailingAction`, `leadingVisual` and `trailingVisual`, a là ActionListItem) makes more sense and is a more flexible solution.

Especially with newer designs considering other actions as at the leading position:

![Screen Shot 2022-11-22 at 18 06 54](https://user-images.githubusercontent.com/293280/203456613-803d69a1-5f63-4523-a1f6-52929049b24e.png)

I think that should clear any confusion between `ParentLink` and `BackButton`, what do you think @broccolinisoup? 🙇

Reply:
Yes, it makes a lot of sense!